schemas/com.snowplowanalytics.snowplow.ecommerce/snowplow_ecommerce_action/jsonschema/1-0-0

```diff
@@ -0,0 +1,37 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Schema for an Ecommerce action",
+    "self": {
+      "vendor": "com.snowplowanalytics.snowplow.ecommerce",
+      "name": "snowplow_ecommerce_action",
+      "format": "jsonschema",
+      "version": "1-0-0"
+    },
+    "type": "object",
+    "properties": {
+      "type": {
+        "description": "The type of the page that was visited E.g. homepage, product page, checkout page.",
+        "enum": [
+          "add_to_cart",
+          "remove_from_cart",
+          "product_view",
+          "list_click",
+          "list_view",
+          "checkout_step",
+          "transaction"
+        ]
+      },
+      "name": {
+        "description": "The name of the list presented to the user E.g. product list, search results, shop the look, frequently bought with.",
+        "type": [
+          "string",
+          "null"
+        ],
+        "maxLength": 128
```

Comment:
The description on the first one doesn't line up with the values - the second one might be fine I'm just not sure on that one :)